lib/client.js

```diff
@@ -190,6 +190,28 @@ class Client {
     return this._paths.some(path => path === '/apis/apps.openshift.io');
   }
 
+  // Adds a namespace to the API request if needed
+  namespacedPath(strings, namespace) {
+    const path = strings[0];  
+    if (namespace === undefined) {
+      return path;
+    }
+    const i = path.lastIndexOf('/'); 
+    return path.slice(0, i) + '/namespaces/' + namespace + path.slice(i, path.length);
+  }
+
+  events(namespace) {
+    return new ApiRequest({ path: this.namespacedPath`/api/v1/events?limit=500${namespace}`, headers: { 'Accept' : 'application/json;as=Table;v=v1beta1;g=meta.k8s.io, application/json' } }, { client: this });
```

Comment:
It should be possible to reuse the `WatchableRequest` version instead of duplicating the method.